src/praisonai/praisonai/cli/interactive/async_tui.py

```
@@ -1194,6 +1232,58 @@ def execute_llm():
+        Collects the working-tree (or staged/per-file) diff via the shipped
+        ``GitManager`` and wraps it in a review rubric fenced as ``diff`` so the
+        read-only ``review`` agent preset can cite exact file:line locations.
+        Returns ``None`` when there is no diff to review so the caller can
+        report a clean working tree.
+        """
+        try:
+            from praisonai_code.cli.features.git_integration import GitManager
+
+            git = GitManager(repo_path=self.config.workspace or None)
+            diff = git.get_diff_content(staged=staged, file_path=file_path)
+        except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)